code/modules/cargo/packs.dm

```diff
@@ -2239,6 +2239,14 @@
 					/mob/living/simple_animal/hostile/retaliate/snake)
 	crate_name = "snake crate"
 
+/datum/supply_pack/critter/lizard
+	name = "Lizard Crate"
+	desc = "Hisss! Containssss a friendly lizard, not to be confusssed with a lizardperssson."
+	cost = CARGO_CRATE_VALUE * 4
+	access_view = ACCESS_JANITOR
```

Comment:
I don't think this has to be janitor only right?

Reply:
This was added in order to keep it consistent with the other pet crates also having access_view (not to be confused with access which locks crates to set access) set to specific jobs (eg. goat to cook, fox to captain, parrots to CE), hence why it's set to janitor. This only affects the IRN tablet program and can be seen by anyone on the cargo supply console/request console.

Reply:
Ah okay, I misunderstood then. Seems good!